pyriemann_qiskit/utils/hyper_params_factory.py

```diff
@@ -1,9 +1,81 @@
-from qiskit.circuit.library import ZZFeatureMap
+from qiskit.circuit.library import ZZFeatureMap, ZFeatureMap, PauliFeatureMap
 from qiskit.algorithms.optimizers import SPSA
 from qiskit.circuit.library import TwoLocal
 import inspect
 
 
+def gen_x_feature_map(reps=2):
+    """Return a callable that generate a XFeatureMap.
+
+    A feature map encodes data into a quantum state.
+    A XFeatureMap is a first-order Pauli-X evolution circuit
+    (no entanglement)
+
+    Parameters
+    ----------
+    reps : int (default 2)
+        The number of repeated circuits, greater or equal to 1.
+
+    Returns
+    -------
+    ret : PauliFeatureMap
+        An instance of PauliFeatureMap
+
```

Comment:
Add reference to Qiskit documentation about `PauliFeatureMap`:
https://docs.quantum.ibm.com/api/qiskit/0.19/qiskit.circuit.library.PauliFeatureMap

Same remark for `gen_z_feature_map` and `ZFeatureMap`.

Reply:
Same remark for `gen_zz_feature_map` and `ZZFeatureMap`.